Resources/Prototypes/Entities/Objects/Specific/Janitorial/soap.yml

```diff
@@ -45,9 +75,20 @@
   description: A Nanotrasen brand bar of soap. Smells of plasma.
   components:
   - type: Sprite
-    state: nt
+    layers:
+    - state: nt-4
+      map: ["enum.SolutionContainerLayers.Fill"]
+  - type: SolutionContainerVisuals
+    fillBaseName: nt-
   - type: Item
     heldPrefix: nt
+  - type: SolutionContainerManager
+    solutions:
+      soap:
+        maxVol: 300
+        reagents:
+        - ReagentId: SoapReagent
+          Quantity: 300
```

Comment:
Made it 50 / 100 / 240.
(tgStation had a comment about how it's 800 since Omega as a greek number has a value of 800, so I did 240 cause it's the 24th letter)